airbyte-integrations/connectors/source-shopify/source_shopify/source.py

```diff
@@ -969,6 +105,15 @@ def test_connection(self) -> tuple[bool, str]:
         except MissingAccessTokenError:
             return False, self.describe_error("missing_token_error")
 
+    def get_shop_id(self) -> str:
+        """
+        We need to have the `shop_id` value available to have it passed elsewhere and fill-in the missing data.
```

Comment:
when do we expect a missing shop_id? Isn't that indicative of a problem that will lead to partial data?

Reply:
Will raise an `Exception` if not obtained. Corrected.